### DISCUSSION TITLE
看看就行 主页（收集下看法

### DISCUSSION THREAD

**Discussion by Joker2184** [Meloong-Git/PCL#4831] [自定义主页]
![QQ截图20240927214008](https://github.com/user-attachments/assets/cf9cd6c0-c3de-4434-a17f-9218e5a2a639)
烂活（
看看就行（
自用的（（
听听看法我改改自己用看着舒服些

**Comment by WorldHim** [comment]
自动更新嘛

**Reply by Joker2184** [reply]
自用暂时还没考虑

**Comment by MoYuan-CN** [comment]
![image](https://github.com/user-attachments/assets/bd69bdcf-a74c-45be-96c5-6657b0be1a7c)
![image](https://github.com/user-attachments/assets/d7e7e197-66d2-4ba1-8471-6df4a0b60811)
扔到单独一张卡片里头去，更新日志应该仅包含日志正文。
不过这种主页应当可以实现自动化？
如有需要帮助可以在信标的主页群或者 @Homeplaza-Of-PCL2 社区找到我

**Reply by Joker2184** [reply]
自动化的（
用信标的构建器跑的（
没感觉和新闻主页一模一样吗（（（（

行
